index.html

```diff
@@ -1,298 +1,287 @@
+          <p>Boostrap</p>
+          <p>Ruby on rails</p>
+          <p>RSpecp</p>
+          <p>Cabybara</p>
+          <p>Selenium</p>
         </div>
-      </div>
-    </section>
-    <!-- Libraries -->
-    <section class="library-container">
-      <ul class="card">
-        <li class="card-item">
-          <img src="./images/icons/Icons1.png" alt="icon" />
-          <h2>Languages</h2>
-          <div class="frameworks">
-            <p>JavaScript</p>
-            <p>HTML</p>
-            <p>CSS</p>
-            <p>Ruby</p>
-          </div>
-        </li>
-        <li class="card-item">
-          <img src="./images/icons/Icons2.png" alt="icon" />
-          <h2>Framworks</h2>
-          <div class="frameworks">
-            <p>Boostrap</p>
-            <p>Ruby on rails</p>
-            <p>RSpecp</p>
-            <p>Cabybara</p>
-            <p>Selenium</p>
-          </div>
-        </li>
-        <li class="card-item">
-          <img src="./images/icons/Icons2.png" alt="icon" />
-          <h2>Skills</h2>
-          <div class="frameworks">
-            <p>Codekit</p>
-            <p>GitHub</p>
-            <p>Codepen</p>
-            <p>Gitlab</p>
-            <p>Terminal</p>
-          </div>
-        </li>
-      </ul>
-    </section>
-    <!-- Form section -->
-    <section class="form" id="contact">
-      <h2>
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
-      </h2>
-      <form action="https://formspree.io/f/xlezroow" method="POST">
-        <input
-          type="text"
-          placeholder="Full name"
-          maxlength="30"
-          name="fullname"
-          required
-        />
-        <input type="email" placeholder="Email address" name="email" required />
-        <textarea
-          name="msg"
-          placeholder="Enter text here"
-          maxlength="500"
-          required
-        ></textarea>
-        <button class="btn" type="submit">Get in touch</button>
-      </form>
-    </section>
+      </li>
+      <li class="card-item">
+        <img src="./images/icons/Icons2.png" alt="icon" />
+        <h3>Skills</h3>
+        <div class="frameworks">
+          <p>Codekit</p>
+          <p>GitHub</p>
+          <p>Codepen</p>
+          <p>Gitlab</p>
+          <p>Terminal</p>
+        </div>
+      </li>
+    </ul>
+  </section>
+  <!-- Form section -->
+  <section class="form" id="contact">
+    <p>
+      I'm always interested in hearing about new projects, so if you'd like to
+      chat please get in touch.
+    </p>
+    <form action="https://formspree.io/f/xlezroow" method="POST">
+      <input type="text" placeholder="Full name" maxlength="30" name="fullname" required />
+      <input type="email" placeholder="Email address" name="email" required />
+      <textarea name="msg" placeholder="Enter text here" maxlength="500" required></textarea>
+      <button class="btn" type="submit">Get in touch</button>
+    </form>
```

Comment:
- [ ] [OPTIONAL] This `input`s can have a `label` attached to them using the `for` and `id` attribute on the `input` element.

Reply:
Ok. noted. Thank you.